src/core/ddsc/tests/whc.c

```diff
@@ -110,6 +110,27 @@ static dds_entity_t create_and_sync_reader(dds_entity_t subscriber, dds_entity_t
   return reader;
 }
 
+static void check_intermediate_whc_state(dds_entity_t writer, seqno_t exp_min, seqno_t exp_max)
```

Comment:
Nit-picking: maybe parameterize `check_whc_state` so that there is less code duplication?

Reply:
Good point